mars/utils.py

```diff
@@ -54,6 +55,7 @@
 import numpy as np
 import pandas as pd
 from sklearn.base import BaseEstimator
+from enum import Enum
```

Comment:
Put imports of built-in packages together. Move below `from contextlib import contextmanager`.

Reply:
done.